2.ServiceTracker_App/App/README.md

```diff
@@ -0,0 +1,101 @@
+## Service Tracker Example
+
+This is an example microservices application with a Javascript Web UI, a MongoDB database, and a series of API microservices. The idea is that various app developers would create Components for their corresponding apps. The overall config will add traits and allow the app to be fully deployed.
+The application being deployed is shown in the following diagram:
+
+![Application architecture diagram](service-tracker-diagram.jpg)
+
+> Full application original source here: https://github.com/chzbrgr71/service-tracker
+
+In this example, there are various roles that handle each aspect of the OAM application.
+
+* UI Developer
+* API Microservices Developer
+* MongoDB Admin
+* App Operator / SRE (handles application deployment in Kubernetes)
+
+## Quickstart: Deploy the sample to Rudr running on any Kubernetes cluster
+
```

Comment:
What's the difference between this README and the one in ServiceTracker_App/README.md?

Reply:
Thanks for finding it. It was duplicate and removed it.